autoinstrumentation/nodejs/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "noImplicitReturns": true,
     "noUnusedLocals": true,
     "pretty": true,
+    "skipLibCheck": true,
```

Comment:
By default, TS does a deep typecheck through transitive dependencies. I don't know if it's a but in opentelemetry-js (/cc @dyladan) but after updating I got a transitive type failure on graphql library.

For our meager code, the transitive type check is definitely not needed.

https://www.typescriptlang.org/tsconfig#skipLibCheck